<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PockerPro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/logo-pocker-pro.png</normaloff>:/img/logo-pocker-pro.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#lnIP[state = &quot;0&quot;]{
	
	background-color: rgb(255, 255, 255);
}

#lnIP[state = &quot;1&quot;]{
	
	background-color: rgb(170, 0, 0);
}

#lnIP[state=&quot;2&quot;]{
	
	background-color: rgb(0, 85, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="signin">
     <widget class="QLabel" name="pockerpro0">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>481</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>POCKER PRO</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>423</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Sign In</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="username">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI Semibold</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="username_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI Semibold</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="signIn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI Semibold</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Sign In</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exit_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI Semibold</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mainpage">
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="pokerprotitle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>461</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>POCKER PRO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="optionsButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>410</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OPTIONS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="creditsButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>490</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CREDITS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>570</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="play">
     <widget class="QLabel" name="playtitle">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hostButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>HOST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="publicButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PUBLIC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="privateButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>490</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PRIVATE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>620</y>
        <width>80</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/back.png);
background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="options">
     <widget class="QLabel" name="pokerprotitle_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>331</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OPTIONS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>620</y>
        <width>80</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/back.png);
background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="volumehorizontalSlider">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>390</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="fullscreen">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>FULLSCREEN</string>
      </property>
     </widget>
     <widget class="QLabel" name="volume">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>290</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>VOLUME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fullscreen_on">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>500</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fullscreen_off">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>500</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="credits">
     <widget class="QLabel" name="creditstitle">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CREDITS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>620</y>
        <width>80</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/back.png);
background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>380</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ant√≥nio Ribeiro
Bernardo Vicente
Guilherme Oliveira
Rui Silva
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Work done by FEUP students from MEEC: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>570</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/logofeup.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="private_opts">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>371</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Connection Information</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblid">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Game ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lnIP">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="state" stdset="0">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnConnect">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="back_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>620</y>
        <width>80</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/back.png);
background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
